buddyboss/buddyboss-platform
# Platform Help section consistent with App Plugin help section

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2376 

### How to test the changes in this Pull Request:

Similar to App Plugin > Provide option to load links within WP sites instead of redirecting to BuddyBoss resources site.
Also, there are differences in the flow between both help sections. Need to make that consistent.
Check Video and related PR below

https://www.loom.com/share/65413327ef3243fcb1316f6239e2be0d

### Proof Screenshots or Video

https://www.loom.com/share/62b83996ce574da7970d22c12f3546d6

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
